### ISSUE TITLE
Empty Screen on Artist Page

### ISSUE THREAD

**Issue by julianschill** [DeMille/headphones-for-musicbrainz#2]
When opening an Artist Page I just get an empty blue screen.
![image](https://cloud.githubusercontent.com/assets/13122110/8436588/a818a6e8-1f59-11e5-8a0a-0196a80ec764.png)

**Comment by DeMille** [comment]
Yeah, musicbrainz just updated their site layout slightly and it broke some of the scraping, just fixed it! Should be up on the chrome store soon,

**Comment by jaredbidlow** [comment]
still see this issue. downloaded the extension from chrome yesterday for the first time.

`Uncaught TypeError: Cannot read property 'innerHTML' of null`
